lib/hako/schedulers/ecs_definition_comparator.rb

```diff
@@ -32,6 +32,7 @@ def definition_schema
           struct.member(:command, Schema::Nullable.new(Schema::OrderedArray.new(Schema::String.new)))
           struct.member(:volumes_from, Schema::UnorderedArray.new(volumes_from_schema))
           struct.member(:user, Schema::Nullable.new(Schema::String.new))
+          struct.member(:privileged, Schema::Nullable.new(Schema::Boolean.new))
```

Comment:
Sorry, didn't check it..

Fixed
https://github.com/eagletmt/hako/pull/20/commits/50b507a484314385d5d44af4739923510a4cf702